Python/venv.ipynb

```diff
@@ -1,4 +1,4 @@
-{
```

Comment:
Missed answer for 4-th question:
Will answer here 
pip install package name
conda install package name